internal/agent/upgrade.go

```diff
@@ -51,8 +52,12 @@ func Upgrade(version, image string, force, debug bool, dirs []string) error {
 			return fmt.Errorf("no releases found")
 		}
 
-		version = releases[0]
-		fmt.Println("latest release is ", version)
+		version = releases[len(releases)-1]
+		fmt.Println("Latest release is ", version)
+		fmt.Printf("Are you sure you want to upgrade to this release? (y/n) ")
+		if !askConfirmation() {
```

Comment:
Looks good! for the prompt maybe we can reuse https://github.com/kairos-io/kairos/blob/master/pkg/utils/console.go#L15

Reply:
Done [here](https://github.com/kairos-io/kairos/pull/389/commits/1d44efef86ac03b50e4ab3d2d9729c180bf12e3a):
```
Latest release is v1.1.6-k3sv1.25.3-k3s1
Are you sure you want to upgrade to this release? (y/n) y 
```